coordinator/tests/services/translation/request/line-item.spec.ts

```diff
@@ -22,13 +22,27 @@ describe("convertMedicationRequestToLineItem", () => {
     firstFhirMedicationRequest = getMedicationRequests(bundle)[0]
   })
 
-  test("Throws TooManyValuesUserFacingError when passed multiple order item numbers", () => {
+  test("Throws TooManyValuesError when passed multiple order item numbers", () => {
     firstFhirMedicationRequest.identifier.push(firstFhirMedicationRequest.identifier[0])
     expect(() =>
       convertMedicationRequestToLineItem(firstFhirMedicationRequest, null, [], [])
     ).toThrow(errors.TooManyValuesError)
   })
 
+  test("Throws TooFewValuesError when passed no dispenseRequest performer-site-type extension", () => {
+    firstFhirMedicationRequest.dispenseRequest.extension = []
+    expect(() =>
+      convertBundleToPrescription(bundle)
+    ).toThrow(errors.TooFewValuesError)
+  })
+
+  test("Throws InvalidValueError when passed no dispenseRequest extensions", () => {
```

Comment:
tried sending an example of this in and got a generic 500 back, is this error being hidden by the proxy?

Reply:
Confirmed that we are receiving 400 for this scenario